specification/http/src/main/scripts/org/kaazing/specification/http/rfc7232/preconditions/if.match/multiple.etags.delete.status.204/response.rpt

```diff
@@ -0,0 +1,29 @@
+property entityTag "\"r2d2xxxx\""
+accept http://localhost:8000/resource
+accepted
+connected
+
+read method "GET"
+read version "HTTP/1.1"
+read closed
+
+write status "200" "OK"
+write header "ETag" ${entityTag}
+write header content-length
+write flush
+write close
+
+accepted
+connected
+
+read method "DELETE"
+read version "HTTP/1.1"
+read header "If-Match" ${entityTag}, "\"tag1\"", "\"tag2\""
+read closed
+
+# One of the etags specified using If-Match header matches the entity-tag of the
+# selected resource. Since the condition evaluates to true, continue to perform
+# the requested method. The response will include 204(No Content) status code.
+write status "204" "No Content"
```

Comment:
Recommend we avoid commenting in the scripts to avoid ambiguity about the source of truth.

All the description should be in the RFC instead and the corresponding ITs can refer to the appropriate section of the RFC in the IT class javadoc comment.